src/metabase/driver/sql_jdbc.clj

```diff
@@ -206,10 +206,24 @@
                                                 :dialect (sql.qp/quote-style driver)))]
       (qp.writeback/execute-write-sql! db-id sql))))
 
+;; kept for get-method driver compatibility
+#_{:clj-kondo/ignore [:deprecated-var]}
 (defmethod driver/alter-columns! :sql-jdbc
   [driver db-id table-name column-definitions]
   (qp.writeback/execute-write-sql! db-id (sql-jdbc.sync/alter-columns-sql driver table-name column-definitions)))
 
+#_{:clj-kondo/ignore [:deprecated-var]}
+(defmethod driver/alter-table-columns! :sql-jdbc
+  [driver db-id table-name column-definitions & opts]
+  (let [deprecated-default-method     (get-method driver/alter-columns! :sql-jdbc)
+        deprecated-driver-method      (get-method driver/alter-columns! driver)
+        deprecated-method-specialised (not (identical? deprecated-default-method deprecated-driver-method))]
```

Comment:
nit: prefer an `?` to end boolean variable.

Otherwise "specialized" here could be taken as an adjective, that's actually my first reading.

Reply:
f2acc2a69b3d

FWIW This is one of those personal style vs project style problems, so will keep an eye out for me doing this. Early in my clj journey I used to put ? on variables/kw flags like java isFoo, isBar, stopped doing that many years ago - reserving it only for predicates. Have to relearn the impulse to `?`. I do understand many projects prefer the ? on everything.